src/server/HtmlDocument.js

```diff
@@ -227,6 +230,19 @@ class HtmlDocument extends React.Component {
             name="msapplication-TileImage"
             content="/static/favicons/mstile-144x144.png"
         />
+
+        <script dangerouslySetInnerHTML={{__html: envConfig}} />
+        { GA_TRACKING_CODE &&
```

Comment:
Right, but it's an inline script block - blocks parsing the page until the JS is done, but doesn't require a network request or anything.